addons/mail/static/src/emoji_picker/emoji_picker.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.EmojiPicker" owl="1">
-    <div class="o-mail-EmojiPicker bg-white" t-att-class="emojis.length === 0 ? 'd-flex flex-column align-items-center justify-content-center' : ''" t-on-click="onClick" t-on-keydown="onKeydown">
+    <div class="o-mail-EmojiPicker bg-white" t-attf-class=" {{props.className}} {{emojis.length === 0 ? 'd-flex flex-column align-items-center justify-content-center' : ''}} {{env.isSmall and env.inChatWindow ? 'o-in-small' : ''}}" t-on-click="onClick" t-on-keydown="onKeydown">
```

Comment:
`o-in-small` => `o-small` (avoid modifiers with dash separator on many words. use camelCase